### DISCUSSION TITLE
Any plans or way to implement this in kubernetes?

### DISCUSSION THREAD

**Discussion by rhmarras** [containrrr/watchtower#739] [Q&A] [answered]
Hi! I like what you've built!
I was wondering if there is a way of implementing this in kubernetes.
Thanks!

**Comment by simonaronsson** [comment, marked as answer]
Hi!

Thank you for the encouragement! At this point, we want to focus on improving the watchtower experience on docker.
Although I've yet to try it myself, https://github.com/keel-hq/keel seems to be doing something similar for k8s.

Best,
Simme

**Reply by XtremeOwnageDotCom** [reply]
Thanks for posting the link for that product. Sure it has helped a few others than myself.
